src/NzbDrone.Core/MediaFiles/EpisodeImport/Augmenting/Augmenters/AugmentQuality.cs

```diff
@@ -0,0 +1,145 @@
+﻿using NLog;
+using NzbDrone.Core.MediaFiles.MediaInfo;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.Qualities;
+using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.MediaFiles.EpisodeImport.Augmenting.Augmenters
+{
+    public class AugmentQuality : IAugmentLocalEpisode
+    {
+        private readonly Logger _logger;
+
+        public AugmentQuality(Logger logger)
+        {
+            _logger = logger;
+        }
+
+        public LocalEpisode Augment(LocalEpisode localEpisode, bool otherFiles)
+        {
+            if (localEpisode.FileEpisodeInfo == null)
+            {
+                return localEpisode;
+            }
+
+            var series = localEpisode.Series;
+            var quality = localEpisode.FileEpisodeInfo.Quality;
+            var downloadClientItemQuality = localEpisode.DownloadClientEpisodeInfo?.Quality;
```

Comment:
I see your point, but we're already using it in other places. We should update the readme, since we're already making use of it and it hasn't tripped anyone up AFAIK.

Reply:
> we're already using it in other places

Wait, yes we are... I thought that wouldn't compile in vs2015. But I was wrong: C# 6.0 _was_ 2015. 2017 is C# 7.x, out variables, tuple deconstruction etc.

So nevermind.